node/child_process_test.ts

```diff
@@ -507,3 +508,91 @@ Deno.test({
     await p;
   },
 });
+
+Deno.test("[node/child_process] spawnSync()", async (t) => {
```

Comment:
Instead of introducing our own tests, can we vendor more of the Node.js test suite for this functionality.

Reply:
Yeah, haha. I thought the same after realising the amount of testing I'd be doing is ridiculous.